test/include/interfacekernels/InterfaceReaction.h

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
You need to add a MOOSE copyright header text here. Please see any of the other source files in the framework for a reference of what this needs to be.